sharding/validator.go

```diff
@@ -31,6 +36,21 @@ func (v *validator) Address() []byte {
 	return v.address
 }
 
+// Chances returns the validator's chances
+func (v *validator) Chances() uint32 {
+	v.mutChances.RLock()
+	defer v.mutChances.RUnlock()
+	return v.chances
+
```

Comment:
delete empty line and also add empy line after defer

Reply:
done